src/assets/styles/app.css

```diff
@@ -6,10 +6,6 @@
     --flow-space: 1.5em;
 }
 
-/** Custom Media Queries **/
-@custom-media --max-content-width-and-up (width >= 65rem);
-@custom-media --less-than-max-content-width (width < 65rem);
```

Comment:
I would suggest keeping these up at the top; they are definitions for the whole file so should be up with the other variable definitions.